content/en/storage-providers/operate/benchmarks.md

```diff
@@ -205,35 +205,202 @@ Available options:
+| `--sector-size`                            | Specify the sector-size (supports: 2K, 512MiB, 32GiB, 64GiB) (default: 512MiB)               |
+| `--miner-addr`                             | pass miner address (only necessary if using existing sectorbuilder) (default: "t01000")      |
+
+### Create sector file
+
+Before we can run sealing benchmarks, we need to create an unsealed sector file. You can specify which sector-size you want to create with the --sector-size flag.
+
+```shell
+./lotus-bench simple addpiece --sector-size <size> /dev/zero /your/path/unsealed
+```
+
+Note that the `unsealed` in `/your/path/` is the file that will be created by the command, and not a directory.
+
+Together with the performance, the command will output a piece CID and the number of bytes in the created unsealed sector. You will need both of these to perform the PreCommit 1 benchmark.
+
+```
+AddPiece 1m26.991655711s (376.7 MiB/s)
+baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq 34359738368
+```
+
+### Sealing tasks
+
+**PreCommit1:**
+
+To run a single PreCommit 1, you will need the CID and amount of bytes in the unsealed sector created in the `addpiece? step. You will also need to specify where the path to the unsealed file is, and where the sealed sector and the cache file will be placed.
 
-### Enabling a custom GPU
+```shell
+./lotus-bench simple precommit1 --sector-size <size> /your/path/unsealed /your/path/sealed /your/path/cache [piece cid] [piece size]
+```
+
+Together with the performance, the command will create a PreCommit1 encoding that you will need if you want to perform the PreCommit 2 benchmark.
 
-If you want to test a GPU that is not explicitly supported, set the following environment variable:
+```
+PreCommit1 3h10m38.942613688s (2.865 MiB/s)
+eyJfbG90dXNfU2VhbFJhbmRvbW5lc3MiOi[...]==
+```
+
+**PreCommit2:**
+
+To run a single PreCommit 2 you will need to specify the path of the sealed sector, the cache file, and the PreCommit1 encoding from the previous step.
 
 ```shell
-export BELLMAN_CUSTOM_GPU="<NAME>:<NUMBER_OF_CORES>"
+./lotus-bench simple precommit2 --sector-size <size> /your/path/sealed /your/path/cache [pc1 encoding]
 ```
 
-Here is an example of trying a GeForce GTX 1660 Ti with 1536 cores:
+Together with the performance, the command will output the `commD` and `commR` that you will need if you want to perform the Commit 1 benchmark.
+
+```
+seal: preCommit phase 2: 14m17.347652736s (38.22 MiB/s)
+d:baga6ea4seaqdsvqopmj2soyhujb72jza76t4wpq5fzifvm3ctz47iyytkewnubq r:bagboea4b5abcbztu2gpgzz746m537wntioqm5mjnfay5dwsugfqyshv4zljmnwyb
+```
+
+**Commit1:**
+
+To run a single Commit 1 you will need to specify the path of the sealed sector, the cache file, and the `commD` and `commR` output from the PreCommit 2 step. You will also need to specify the path for the json output from the Commit1 step.
```

Comment:
```suggestion
To run a single Commit1 you will need to specify the path of the sealed sector, the cache file, and the `commD` and `commR` output from the PreCommit2 step. You will also need to specify the path for the json output from the Commit1 step.
```
This seems to change halfway through the page. Would it be better to keep the format consistent?